src/NuGet.Clients/NuGet.PackageManagement.UI/PackageLicenseUtilities.cs

```diff
@@ -42,6 +43,20 @@ internal static IReadOnlyList<IText> GenerateLicenseLinks(LicenseMetadata licens
             return new List<IText>();
         }
 
+        internal static Paragraph[] GenerateParagraphs(string licenseContent)
+        {
+            var textParagraphs = licenseContent.Split(
+                new[] { "\n\n", "\r\n\r\n" }, // Take care of paragraphs regardless of the name ending. It's a best effort, so weird line ending combinations might not work too well.
```

Comment:
I don't love this, but if we need paragraphs I don't know of a best way to do it.